.travis.yml

```diff
@@ -0,0 +1,4 @@
+language: node_js
+node_js:
+  - "8"
+  - "4"
```

Comment:
No 6?

Reply:
Good catch, doh
Personally I'd like to use "LTS" but IIRC nvm's author regrets some other notations and doesn't want to enable that one
I'm thinking the goals are: keep current with a little bit of backwards compatibility but requiring minimal changes/manual maintenance

Reply:
[Looks like `lts/argon`, `lts/boron`, `lts/*` etc are now valid](https://github.com/creationix/nvm/pull/1070#issuecomment-217266648)
Though now I'm wondering what the first two would avail us over the major version numbers

IDK that
```
node_js
  - "8"
  - "lts/*"
  - "4"
```
makes too much sense, so maybe using the major number and updating every LTS event is the best way?

Reply:
Added 6 and rebased to make merging easier (even though GitHub allows for multiple merge strategies now)